src/Aspire.Cli/DotNetCliRunner.cs

```diff
@@ -29,6 +29,30 @@ public async Task<int> RunAsync(FileInfo projectFile, bool watch, string[] args,
             cancellationToken: cancellationToken);
     }
 
+    public async Task<int> CheckHttpCertificateAsync(CancellationToken cancellationToken)
+    {
+        string[] cliArgs = ["dev-certs", "https", "--check"];
+        return await ExecuteAsync(
+            args: cliArgs,
+            env: null,
+            workingDirectory: new DirectoryInfo(Environment.CurrentDirectory),
+            backchannelCompletionSource: null,
+            streamsCallback: null,
+            cancellationToken: cancellationToken);
+    }
+
+    public async Task<int> TrustHttpCertificateAsync(CancellationToken cancellationToken)
+    {
+        string[] cliArgs = ["dev-certs", "https", "--trust"];
```

Comment:
On Linux, it's also required that an environment variable be set (`SSL_CERT_DIR`) to fully trust the certificate. See https://learn.microsoft.com/aspnet/core/security/enforcing-ssl?view=aspnetcore-9.0&tabs=visual-studio%2Clinux-sles#linux-specific-considerations for more info

Reply:
This is good info. I wonder how we can communicate this to the developer. I don't think that there is any meaningful way we can update the users machine to set this environment variable unless we want to get into the business of detecting and appending to users scripting profiles.